test/plugins/minifyStyles.13.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 48 48">
+    <defs>
+        <style>/* test */</style>
```

Comment:
At present, the test here fails, and it actually generates:
```svg
<style>
</style >
```
I don't know how to solve it yet.